packages/slate-cli/src/utils.js

```diff
@@ -140,6 +140,25 @@ export function hasDependency(dependencyName, pkg) {
   return hasDependencies;
 }
 
+
+/**
+ * Beautify a JSON file
+ *
+ * @param {string} file - The path to the file.
+ */
+export function beautifyJson(file) {
```

Comment:
I feel like `unminifyJson` assumes that the file is minified — which it might not be — though I don't hold to that opinion very strongly. `unminifyJson` works for me.